### PR TITLE
[5.1] Support : add copy and move functions to Collection

### PR DESCRIPTION
**Pull request by lucasmichot**

### CODE REVIEW

File: src/Illuminate/Support/Collection.php [hunk 1]
```diff
@@ -161,6 +161,46 @@ public function where($key, $value, $strict = true)
...
+	 * @return static
+	 */
+	public function move($from, $to)
+	{
+		$to = is_array($to) ? $to : [$to];
+
+		foreach ($to as $dest)
+		{
+			$this->items[$dest] = $this->items[$from];
+		}
```
Comment (franzliedke) [inline src/Illuminate/Support/Collection.php]: You could simply use `copy()` here...

File: src/Illuminate/Support/Collection.php [hunk 2]
```diff
@@ -161,6 +161,46 @@ public function where($key, $value, $strict = true)
...
 	/**
+	 * Copy an item of the collection.
+	 *
+	 * @param  mixed  $from
+	 * @param  mixed  $to
+	 * @return static
+	 */
+	public function copy($from, $to)
+	{
+		$to = is_array($to) ? $to : [$to];
```
Comment (franzliedke) [inline src/Illuminate/Support/Collection.php]: I think `$to = (array) $to;` would work, too.

**Comment by taylorotwell** [comment]
I think we are maybe getting a bit carried away with Collection.

**Comment by GrahamCampbell** [comment]
Yeh, I agree. Arrays are fine, and use less memory and cpu time. :)

**Pull request closed without merging**